example/android/app/build.gradle

```diff
@@ -8,15 +8,15 @@ plugins {
 android {
     namespace = "net.jonhanson.flutter_native_splash_example"
     compileSdk = flutter.compileSdkVersion
-    ndkVersion = flutter.ndkVersion
+    ndkVersion = "27.0.12077973"
```

Comment:
Hi, thanks for this PR.  One question: Why does this use a specific NDK version instead of sticking to Flutter's setting of the NDK version?

Reply:
A warning suggesting to use the latest NDK version pops up for me when running the example with the latest flutter version.
But because it is only within the example, I wouldn't mind dropping this change.
My personal priority would be to get rid of the warning mentioning the deprecated Java source and target compatibility versions.

Reply:
Can you remove the NDK line and then I will merge it in.

Reply:
Done.